src/_blake2.ts

```diff
@@ -2,7 +2,7 @@ import assert from './_assert.js';
 import { Hash, Input, toBytes, u32 } from './utils.js';
 // For BLAKE2b, the two extra permutations for rounds 10 and 11 are SIGMA[10..11] = SIGMA[0..1].
 // prettier-ignore
-export const SIGMA = new Uint8Array([
+export const SIGMA = /*#__PURE__*/ new Uint8Array([
```

Comment:
Should we change it to `@__PURE__`? I think @ was used in other places.

Reply:
it can be both `#` and `@` https://github.com/terser/terser#annotations

[babel-plugin-annotate-pure-calls](https://github.com/Andarist/babel-plugin-annotate-pure-calls) uses `#`. I didn't change this. can unify all annotations to one type. which one do you like more `#` or `@`?

Reply:
`@`